check_galera_cluster

```diff
@@ -43,6 +43,29 @@ print_help() {
 crit=2
 fcp=0.1
 
+#check required tools
+check_executable() {
+
+    if [ -z "$1" ]; then
+        echo "check_executable: no parameter given!"
+        exit $ST_UK
+    fi
+
+    if [ -z "$(which $1)" ]; then
+            echo "UNKNOWN: Cannot find $1"
+            exit $ST_UK
+    fi
+
+}
+check_executable mysql
+check_executable bc
+
+
+if (( $# == 0 )); then
```

Comment:
i'm not sure about that. If i remember correctly, you can use the script with no arg at all, if you have a mysql client config like `.my.cnf`, so mysql can connect without specific args like password/user/host/...